sdk/search/Azure.Search.Documents/tests/Utilities/SearchTestBase.cs

```diff
@@ -266,7 +266,7 @@ static void AssertEqual(object e, object a, string path)
         protected async Task WaitForDocumentCountAsync(
             SearchClient searchClient,
             int minimumCount,
-            int attempts = 3,
+            int attempts = 10,
             TimeSpan? delay = null)
```

Comment:
Can we quickly check all the callers of this API to make sure nobody's passing in a delay that's going to be crazy if when multiplied by 10 instead of just 3?

Reply:
Yes, I checked the callers. None of them explicitly set the `attempts` parameter.